platform/android/java/lib/src/org/godotengine/godot/plugin/GodotPlugin.java

```diff
@@ -71,9 +71,9 @@
  * - 'plugin.init.ClassFullName' is the full name (package + class name) of the plugin class
  * extending {@link GodotPlugin}.
  *
- * A plugin can also define and provide c/c++ gdnative libraries and nativescripts for the target
+ * A plugin can also define and provide c/c++ gdextension libraries and nativescripts for the target
  * app/game to leverage.
- * The shared library for the gdnative library will be automatically bundled by the aar build
+ * The shared library for the gdextension library will be automatically bundled by the aar build
  * system.
  * Godot '*.gdnlib' and '*.gdns' resource files must however be manually defined in the project
  * 'assets' directory. The recommended path for these resources in the 'assets' directory should be:
```

Comment:
To be thorough, the comments will need to be updated as well to match the gdextension workflow (e.g: replace occurrences of `gdnlib` and `gdns` with `gdextension`).